schema/Retail/RET/on_init.ts

```diff
@@ -528,7 +528,6 @@ export const onInitSchema = {
                 '@ondc/org/settlement_details',
               ],
             },
-
             tags: {
               type: 'array',
               items: {
```

Comment:
provider_tax_number and tax_number must be made mandatory in /tags/bpp_terms

Reply:
done